manager/state/raft/raft.go

```diff
@@ -342,6 +398,9 @@ func (n *Node) JoinAndStart(ctx context.Context) (err error) {
 	}
 
 	// join to existing cluster
+	if n.opts.Addr == "" {
```

Comment:
I don't really understand events sequence. What ensures that `BindRemote` is called before `Run`?